Client/mods/deathmatch/logic/CStaticFunctionDefinitions.cpp

```diff
@@ -3900,6 +3900,22 @@ bool CStaticFunctionDefinitions::GetObjectElasticity(CClientObject& Object, floa
     return true;
 }
 
+bool CStaticFunctionDefinitions::GetRopeHeight(CClientObject& Object, float& height)
```

Comment:
The first argument should be a const reference. Just because we want to get a value, but not change anything inside **Object**.